addons/website_sale/views/templates.xml

```diff
@@ -1381,102 +1369,90 @@
+                                    t-att-alt="line.name_short"/>
+                            </span>
+                            <span t-else=""
+                                t-field="line.product_id.image_128"
+                                t-options="{'widget': 'image', 'qweb_img_responsive': False, 'class': 'rounded o_image_64_max'}"
+                            />
+                        </td>
+                        <td t-attf-class="td-product_name {{ 'pt-4' if line_index &gt; 0 else ''}}" data-cell-type="product_name">
+                            <t t-call="website_sale.cart_line_product_link">
+                                <strong t-field="line.name_short" class="d-inline-block align-top"/>
+                            </t>
+                            <t t-call="website_sale.cart_line_description_following_lines">
+                                <t t-set="div_class" t-value="'d-none d-md-block'"/>
+                            </t>
+                        </td>
+                    </t>
+                    <td t-attf-class="text-center td-qty pe-0 {{ 'pt-4' if line_index &gt; 0 else ''}}">
+                        <div class="css_quantity input-group mx-auto float-end">
+                            <t t-if="not line._is_not_sellable_line()">
+                                <t t-if="show_qty">
+                                    <a t-attf-href="#" class="btn btn-link js_add_cart_json d-inline-block border-end-0" aria-label="Remove one" title="Remove one">
+                                        <i class="fa fa-minus"></i>
+                                    </a>
+                                    <input type="text" class="js_quantity form-control quantity border-start-0 border-end-0" t-att-data-line-id="line.id" t-att-data-product-id="line.product_id.id" t-att-value="int(line.product_uom_qty) == line.product_uom_qty and int(line.product_uom_qty) or line.product_uom_qty" />
+                                    <t t-if="line._get_shop_warning(clear=False)">
+                                        <a t-attf-href="#" class="btn btn-link">
+                                        <i class='fa fa-warning text-warning' t-att-title="line._get_shop_warning()" role="img" aria-label="Warning"/>
                                         </a>
                                     </t>
-                                    <t t-else="">
-                                        <input type="hidden" class="js_quantity form-control quantity" t-att-data-line-id="line.id" t-att-data-product-id="line.product_id.id" t-att-value="int(line.product_uom_qty) == line.product_uom_qty and int(line.product_uom_qty) or line.product_uom_qty" />
-                                    </t>
+                                    <a t-else='' t-attf-href="#" class="btn btn-link float_left js_add_cart_json d-inline-block border-start-0" aria-label="Add one" title="Add one">
+                                        <i class="fa fa-plus fs-small"></i>
```

Comment:
I don't think that `fs-small` exists 😉
You probably needed `small` , remember to check the result on screen after applying
```suggestion
                                        <i class="fa fa-plus small"/>
```